### ISSUE TITLE
Panic when paths is an array

### ISSUE THREAD

**Issue by hugoboos** [pb33f/libopenapi#4]
Given the two documents:

```json
{
    "openapi": "3.1.0",
    "paths": [
        {"/": {
            "get": {}
        }}
    ]
}
```

```json
{
    "openapi": "3.1.0",
    "paths": [
        "/": {
            "get": {}
        }
    ]
}
```

```go
import (
	"github.com/pb33f/libopenapi"  
	oahigh "github.com/pb33f/libopenapi/datamodel/high/v3"
	oalow "github.com/pb33f/libopenapi/datamodel/low/v3"
)

oas, err := libopenapi.NewDocument(spec)
if err != nil {
	return err
}

if doc, errs := oalow.CreateDocument(oas.GetSpecInfo()); errs == nil {
	oad := oahigh.NewDocument(doc)
}
````

The call to `oalow.CreateDocument` causes the following panic: `panic: runtime error: index out of range [1] with length 1`

Stacktrace:
```
goroutine 26 [running]:
github.com/pb33f/libopenapi/index.(*SpecIndex).GetOperationsParameterCount(0x140000a0c00)
	/go/pkg/mod/github.com/pb33f/libopenapi@v0.1.7/index/spec_index.go:1342 +0x142c
github.com/pb33f/libopenapi/index.runIndexFunction.func1(0x0?, 0x0?)
	/go/pkg/mod/github.com/pb33f/libopenapi@v0.1.7/index/spec_index.go:185 +0x28
created by github.com/pb33f/libopenapi/index.runIndexFunction
	/go/pkg/mod/github.com/pb33f/libopenapi@v0.1.7/index/spec_index.go:184 +0x30
```

**Comment by daveshanley** [comment]
Whoops! nice catch. I will fix this ASAP.

**Comment by daveshanley** [comment]
This has been fixed in v0.1.10. I appreciate your help in making this a better library.
